Doc/library/winreg.rst

```diff
@@ -278,6 +278,26 @@ This module offers the following functions:
       isn't.
 
 
+.. function:: GetValue(key, sub_key, value_name)
+
+   Retrieves the type and data for a specified value name associated with
+   an open registry key.
+
+   *key* is an already open key, or one of the predefined
+   :ref:`HKEY_* constants <hkey-constants>`.
+
+   *sub_key* is a string that holds the name of the subkey with which the value is
+   associated.  If this parameter is ``None`` or empty, the value will be read from *key*.
+
+   *value_name* is a string indicating the value to query.
+
+   The result is a tuple of 2 items, the same as :meth:`QueryValueEx`
```

Comment:
It should be noted that `GetValue()` automatically expands environment variables in `REG_EXPAND_SZ` strings and returns the type as `REG_SZ`. For example:
```python
>>> QueryValueEx(HKEY_CURRENT_USER, 'test')
('%SystemRoot%', 2)
>>> GetValue(HKEY_CURRENT_USER, '', 'test')
('C:\\Windows', 1)
```
This default behavior, which depends on the current user's environment, is potentially very wrong for `REG_EXPAND_SZ` values in a system key or for another user in `HKEY_USERS`.

If flags are supported, the above should be noted as the default behavior, which can be disabled with `RRF_NOEXPAND`. For example:

```python
>>> GetValue(HKEY_CURRENT_USER, '', 'test', RRF_RT_ANY | RRF_NOEXPAND)
('%SystemRoot%', 2)
```
Maybe for supporting flags if `!(flags & RRF_RT_ANY)`, it can simply use `flags | RRF_RT_ANY`. It's clear that the low word is reserved for data types. This way the caller just needs to pass a high word flag such as `RRF_NOEXPAND`, `RRF_SUBKEY_WOW6432KEY`, etc.